apps/src/blocklyAddons/cdoWorkspaceSvg.js

```diff
@@ -27,6 +27,7 @@ export default class WorkspaceSvg extends GoogleBlockly.WorkspaceSvg {
     return false; // TODO
   }
   setEnableToolbox() {} // TODO
+  traceOn() {} // TODO
```

Comment:
this method was deprecated in ~2016, and was just deleted in the latest release. TODO: figure out what it used to do and if we need to re-implement it. Otherwise, we can just leave it as a no-op for now and delete the usage in our codebase when we finish migrating to Google blockly.